### DISCUSSION TITLE
Welcome to cross-compile-godot-raspberry-pi Discussions!

### DISCUSSION THREAD

**Discussion by hiulit** [hiulit/cross-compile-godot-raspberry-pi#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Burnsedia** [comment]
I am thinking about making a container so it is easier for non Linux users or people using a non-gnu Linux distro to cross-compile
